javascript/ql/lib/change-notes/2023-04-13-Forge-truncated-sha512-hash

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* The Forge module in `CryptoLibraries.qll` now correctly classifies SHA-512/224,
+* SHA-512/256, and SHA-512/384 hashes used in message digests as NonKeyCiphers.
```

Comment:
```suggestion
  SHA-512/256, and SHA-512/384 hashes used in message digests as NonKeyCiphers.
```
(Remove the second bullet point since this is a single sentence)

Also could you rename this file to have the `.md` extension, and move it to `ql/src/change-notes` instead?

Reply:
Sure.  Why would my change note go in the src/changenotes folder when I changed the CryptoLibraries.qll file in the lib folder?